Unable to install since helm 3.18.0

## UPSTREAM ISSUE

* https://github.com/helm/helm/issues/30876

### Current Behavior

```
$ helm plugin install https://github.com/jkroepke/helm-secrets
Error: plugin is installed but unusable: both platformCommand and command are set in "/home/runner/.local/share/helm/plugins/helm-secrets/plugin.yaml"
Error: Process completed with exit code 1.
```

### Expected Behavior

No issues when installing

### Steps To Reproduce

```markdown
1. Install helm 3.18.0
2. install this plugin
```

### Environment

- Helm Version: 3.18.0
- Helm Secrets Version: latest
- ArgoCD Version
- OS:
- Shell:


### Anything else?

_No response_

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+
+## [4.6.5] - 2025-05-20
+
+### Fixes
+- fix: Unable to install since helm 3.18.0 #530 (https://github.com/jkroepke/helm-secrets/pull/531)
+
 ## [4.6.4] - 2025-05-02
 
 ### Fixes

--- a/plugin.yaml
+++ b/plugin.yaml
@@ -5,7 +5,10 @@ description: |-
   This plugin provides secrets values encryption for Helm charts secure storing
 useTunnel: false
 platformCommand:
-  - command: "$HELM_PLUGIN_DIR/scripts/run.sh"
+  - os: linux
+    command: "$HELM_PLUGIN_DIR/scripts/run.sh"
+  - os: darwin
+    command: "$HELM_PLUGIN_DIR/scripts/run.sh"
   - os: windows
     command: >-
       cmd /c $HELM_PLUGIN_DIR\scripts\wrapper\run.cmd

--- a/plugin.yaml
+++ b/plugin.yaml
@@ -4,8 +4,8 @@ usage: "Secrets encryption in Helm for Git storing"
 description: |-
   This plugin provides secrets values encryption for Helm charts secure storing
 useTunnel: false
-command: "$HELM_PLUGIN_DIR/scripts/run.sh"
 platformCommand:
+  - command: "$HELM_PLUGIN_DIR/scripts/run.sh"
   - os: windows
     command: >-
       cmd /c $HELM_PLUGIN_DIR\scripts\wrapper\run.cmd

